.github/workflows/ci.yml

```diff
@@ -834,7 +834,7 @@ jobs:
     timeout-minutes: 120
     name: "Wait for PROD images"
     runs-on: ubuntu-20.04
-    needs: [build-info]
+    needs: [build-info, ci-images]
```

Comment:
Add a comment saying why we are doing this please, becuase we _might_ be able to reverse this once we have self-hosted runners set up

Reply:
I could add it, but I think this is a 'target' optimization. Even if  we switch to self-hosted runners, we want to keep  it in.

Currently our dependency model looks like:

1) Workflow run  builds CI image -> once done CI Iimage waiting finishes and CI tests run

2) in Worklflow Run PROD images are build using the CI images from step 1 (this is needed to build provider packages because prod images are build from packages now). 

Step 2) depends on 1), so there will never be the case that PROD images will be available before CI images.. So waiting for PROD images in parallel to CI images simply makes no sense.